### DISCUSSION TITLE
I Just Have A Reccomendation

### DISCUSSION THREAD

**Discussion by thisismyaltaccountt** [theshmunguss1/RamOS#14] [Ideas]
Hi! I know this is a beta, but I noticed that when I opened the website and opened an app, it just redirected me to a new page. Maybe instead of doing that, you could just make little windows pop up that are iframing the other pages that you could make a div element for the apps and just make it appear/hide, because it's a little clunky when you redirect someone to another page. So far I really like the idea! Good luck! I would reccomend looking at the code for other things like Duck OS to get some ideas on how to do this.

**Comment by DJshelfmushroom** [comment]
This is planned in the future, if you would like to contribute, fork it and make a pull request!

**Comment by thisismyaltaccountt** [comment]
I'l try to find time to contribute! (on my main account)

**Reply by DJshelfmushroom** [reply]
lol

**Reply by theshmunguss1** [reply]
Could you drop your main account here?

**Reply by TrapjawwS** [reply]
Here I am. (I looked into making the windows, but I've only been using JS for about a year, so it might be a bit of a struggle to get it working, but I can try!)

**Reply by DJshelfmushroom** [reply]
check out https://jqueryui.com/dialog/ for some ideas
